### ISSUE TITLE
How is maintenance going?

### ISSUE THREAD

**Issue by rugk** [SilenceIM/Silence#612]
Silence [has not received any commits/updates](https://github.com/SilenceIM/Silence/graphs/commit-activity) since May.

We currently have a [very](https://github.com/SilenceIM/Silence/issues/611) [buggy](https://github.com/SilenceIM/Silence/issues/580) [unstable version](https://github.com/SilenceIM/Silence/issues/555), whcih FDroid users [accidentally received](https://github.com/SilenceIM/Silence/issues/596) as a "stable version update".

The only thing, I want, is a **stable, nice, tiny and secure Silence**. [Adding XMPP](https://github.com/SilenceIM/Silence/pull/390) is really not necessary and seeing the status of your development, it really looks like [feature creep](https://en.wikipedia.org/wiki/Feature_creep). I am sorry, but well…
As you already put your effort into it, no problem, merge it. Some people may like it (and already expressed that) If you can maintain it…

If no, better **focus on the Silence we know and love :heart:**! Stick to your [project goals](https://github.com/SilenceIM/Silence#project-goals) ("keep SMS/MMS encryption", "merge upstream security fixes") and your [feature list](https://github.com/SilenceIM/Silence#user-content-silence-), which gets it to the point accurately: **Easy, Reliable, Private, Safe, Open Source**.

People may like XMPP, but one thing is for sure: No one here would like it to see Silence to went away.

**Comment by rugk** [comment]
When writing this, I saw https://github.com/SilenceIM/Silence/issues/610. So it's good something is ongoing there. Anyway, I'd still send this statement here. May it encourage you! :+1:

**Comment by bugith** [comment]
Please, don't forget #130 because it is the single reason I don't use Silence instead of the defauft messages app. I don't care at a first glance if I can't sent encrypted SMS to a group. The problem is some people don't have a phone plan with MMS (or they are expensive, so they disable DATA). 
As you said : "This functionality is partially present in the code. When we forked TextSecure, group messages were only push messages or MMS messages. I worked on a way to send encrypted group SMS messages, but the code needed a clean-up. Today, I'm focus on #390, but I will work on group SMS functionality right after."
Now you've dropped #390 , I hope will have group sms ! .... soon ?
Thank you man

**Comment by ArchangeGabriel** [comment]
Where did you see #390 was dropped? It’s not AFAIK.

**Comment by bugith** [comment]
Hi
The last but one post in #390 on sept 17, 2017 (as of oct 21st, 2017) directs to the OP of this very #612 that states "no commits/updates since May (2017, Ed.)". Did I misunderstand something, as a non-english alien ?
What ever about xmpp support, it is an extension to the SMS and MMS in Silence that should not stop trials to support traditional features, i.e. group SMS (unless the goal is to drop non-IP transports, then SMS and MMS themselves). Is it the goal, or at least an anticipation of a short|medium|long-term general trend ?

**Comment by ArchangeGabriel** [comment]
@bugith rugk is not a Silence developer, just an user expressing his opinion (neither am I btw). That’s likely what mislead you.

Regarding XMPP, no, the goal is not to replace and thus drop SMS and MMS ever, Silence was created initially because Signal (formerly TextSecure) dropped support for SMS. XMPP is intended for people having IP connectivity and wanting to also hide metadata, which is very important. But Silence still wants to take into account users without IP connectivity and thus relying on sole SMS for messaging purpose.

In this regard group SMS is an important feature, but given that I’ve never seen any application or ROM supporting them, I’m not sure what are the difficulties here.

**Comment by johanw666** [comment]
And don't forget people who live somewhere were data might be blocked and sms will not (China, Iran?) SMS is an importaant backup channel in such cases, and encryption might be very important. Further, there are people like me who use providers who are or already have phased out MMS completely. In Europe, MMS was always very expensive and unpopular so some providers are abandoning it because WhatsApp has de facto replaced it.

**Comment by bugith** [comment]
@ArchangeGabriel , thanks for clarifications. About group sms, maybe we don't speak about the same thing : for me, and surely this is a bad use of the word, this what I do with any app that is able to when I send a sms to several contacts in the same operation, i.e. when I selected them one by one in my contacts book. In fact IIRC I never found an app able to select a group and sent a message to.
But I only said my hope to be able to send a sms to 2+ users in 1 op with Silence, even with no cypher/no attachment ... we must keep feature requests for later ;-)

**Comment by ArchangeGabriel** [comment]
All the app I’ve ever tried always used MMS for sending SMS to multiple users.

**Comment by bugith** [comment]
Did you ever tried the stock messaging app ? The one in a samsung n7505 send sms to multiple users : I just checked, disabling DATA & Wifi and the recipents got it

**Comment by ArchangeGabriel** [comment]
Yes, I’ve used stock SMS app before being on Silence. But everytime I’ve tried to send SMS to multiple users, it was automatically converted to MMS.

I’ve just asked two people around me still using the stock SMS app, and it converts to MMS despite no network being available (and thus the message isn’t even sent).

**Comment by bugith** [comment]
I don't talk about only android : even before android I could send to multi recipient with the standard SMS system, the one that uses the signalization channels of the GSM protocol.
The only ~limitation~ was the piece of soft in the phones firmware didn't allow to reply to group. That's easy to understand why : SMS being limited to 140/8b (or 160/7b or 70/16b), adding an undefined lentgh number of recipients to any of them (so that it could have be able to "reply to group") could have shorten the text payload to length.... to less than zero :)
Maybe our respective suppliers play around with tcp/ip in the back ground to emulate thee old sms or is it possible they can ignore both our setting "no mobile data" + "no wifi" ?

**Comment by rugk** [comment]
As a fallback you could always just send a single SMS to each phone number. So it works more like a distribution/mailing list. There should be a warning, but for users with unlimited SMS plan, this is no problem. (And that becomes quite common, in contrast to MMS, which can still be very costly.)

**Comment by pR0Ps** [comment]
This isn't really the place for this, please move the discussion to the
actual issue at hand ( https://github.com/SilenceIM/Silence/issues/130 )
